### PR TITLE
Load pt tanh node

### PR DESCRIPTION
**Pull request by jackm321**
Summary:
Load `aten::tanh` and inplace version `aten::tanh_`.

Documentation:
doxygen

Test Plan:
new unit test

### CODE REVIEW

File: torch_glow/src/PyTorchModelLoader.cpp [hunk 1]
```diff
@@ -1259,6 +1260,19 @@ Error PyTorchModelLoader::loadRelu(const torch::jit::Node *ptNode) {
...
 }
 
+Error PyTorchModelLoader::loadTanh(const torch::jit::Node *ptNode) {
+  auto inputs = ptNode->inputs();
+  auto outputs = ptNode->outputs();
+  RETURN_IF_ERR(checkInputAndOutputSizes(inputs, 1, outputs, 1));
+
+  glow::NodeValue input;
+  ASSIGN_VALUE_OR_RETURN_ERR(input, getGlowNodeValueForValue(inputs[0]));
+  input = rescaleUIntToInt(input);
```
Comment (yinghai) [inline torch_glow/src/PyTorchModelLoader.cpp]: Why do we do this?

Comment (jackm321) [reply torch_glow/src/PyTorchModelLoader.cpp]: Oh it's for quantized inputs, I copied it accidentally grabbed it from relu. I'll remove it and if someone needs it for quantized op they can add it later

File: torch_glow/src/PyTorchModelLoader.cpp [hunk 2]
```diff
@@ -1259,6 +1260,19 @@ Error PyTorchModelLoader::loadRelu(const torch::jit::Node *ptNode) {
...
+
+  glow::TanhNode *glowNode = F_.createTanh("tanh", input);
+  return addValueMapping(outputs[0], rescaleIntToUint(glowNode->getResult()));
```
Comment (yinghai) [inline torch_glow/src/PyTorchModelLoader.cpp]: Yeah, also here.

**Comment by jackm321** [comment]
@yinghai what do you mean? If it's the uint8 requantization thing, it only runs on quantized inputs

**Comment by yinghai** [comment]
@jackm321  land this?

**Pull request closed without merging**